reactioncommerce/reaction
# Add email updated translation

Resolve #3307 

- Added email updated translation
- Whenever user try to update email it shows `accountUI.info.emailUpdated` instead of `emailUpdated`

#### Test
- Login as Admin or User
- Click  profile to reveal the profile page.
- Change email and click on update email address.
- Notice the toastr message 'Email Updated'.
